contributing/guidelines.rst

```diff
@@ -11,7 +11,7 @@ PHP Style
 =========
 
 All code must conform to our `Style Guide
-<./styleguide.rst>`_, which is
+<./styleguide.md>`_, which is
 essentially the `Allman indent style
```

Comment:
This should no longer be Allman, I think. Perhaps, the K&R style now applies.

Reply:
Can we just say "PSR-12" instead?

Reply:
Are you going to use this PR to correct items that are not related to the purpose of the PR?

Reply:
@paulbalandan @MGatner I will update the entire Contribution Guidelines in another PR.

Reply:
We have two Guidelines:
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/CONTRIBUTING.md
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/guidelines.rst

It seems it is better to merge into one (CONTRIBUTING.md).

Reply:
No, instead of merging them into one, it might be better to change them to eliminate duplication of content.